crucible-saw/src/Lang/Crucible/Backend/SAWCore.hs

```diff
@@ -191,10 +191,11 @@ bindSAWTerm sym bt t = do
   return sbVar
 
 newSAWCoreBackend ::
```

Comment:
@travitch am I right assuming `newSawCoreBackend` should take the float mode flag too? I ask because the current API is fixing the SAWCore solver as Yices, so it wouldn't be _crazy_ to keep it also tied to the real interpretation mode for now I guess... I could be convinced either way ¯\\_(ツ)\_/¯

Reply:
Oops, I think I misspoke a little here, sorry. Although this function is pinned at Yices, you could still annotate whatever float flag you wanted previously, so pinning it at `FloatReal` is _not_ backwards compatible =\ I think the simplest/sanest thing is to just have the flag as I originally had it.

Reply:
It's probably worth noting that whatever you do here is unlikely to break anything. We currently have no way to write specifications of programs that use floats (though that's coming soon!), so we aren't really using this aspect of What4 in any visible way. Symbolic floats or reals can't be created in the normal SAW workflow.